ui/apps/platform/src/routePaths.js

```diff
@@ -159,7 +160,7 @@ export const basePathToLabelMap = {
     [policyManagementBasePath]: 'Policy Management',
     [policiesBasePath]: 'Policy Management',
     [policyCategoriesPath]: 'Policy Categories',
-    [collectionsPath]: 'Collections',
+    [collectionsBasePath]: 'Collections',
```

Comment:
As a follow up, let’s discuss page title in browser history and tab.

Unlike path, which must be global, containers can take responsibility for page title.

Reply:
Yes good call, I'll create a Jira task to address titles before I merge this PR.

Reply:
https://issues.redhat.com/browse/ROX-12952 ✅